R/lockfile.R

```diff
@@ -116,16 +116,24 @@ renv_lockfile_path <- function(project) {
   renv_paths_lockfile(project = project)
 }
 
+renv_manifest_path <- function(project = NULL) {
+
+  # similar to default case of `renv_paths_lockfile()`
+  project <- renv_project_resolve(project)
+  renv <- renv_paths_renv(project = project)
+  file.path(dirname(renv), "manifest.json")
```

Comment:
Was unsure of the exact semantics here; I just followed the `renv_paths_lockfile()` function.

Reply:
I believe you just want `file.path(project, "manifest.json")` here? If that's the case, you can just remove this function too.

Reply:
Yeah, that really is all I want. :)

Reply:
Yeah, that really is all I want. :) I wasn't sure if `renv_resolve_project()` would account for anything where the actual dir needed for `project` was different from the string passed in (I assumed that was why it was called in the codepath that resolves the lockfile).

I'll remove the function :)